src/Logary/Configuration/Events.fs

```diff
@@ -31,23 +31,33 @@ module Events =
       fun (msgs: #seq<_>) ->
         HasResult << Alt.prepareJob <| fun () ->
 
+        let putAllPromises = IVar ()
+
         msgs
-        |> Seq.Con.mapJob (fun msg -> next msg |> PipeResult.orDefault LogResult.rejected)
-        |> Job.map (fun results -> Result.sequence (results.ToArray()))
-        |> Job.map (function
-          | Ok _ ->
-            LogResult.success
-          | Result.Error [] ->
-            failwithf "No results for %A" (List.ofSeq msgs)
-          | Result.Error (e :: _) ->
-            Alt.always (Result.Error e))
-        )
+        |> Seq.Con.mapJob (fun msg -> next msg |> PipeResult.orDefault LogResult.success)
+        >>= IVar.fill putAllPromises
```

Comment:
Why not just bind `reduce` as the callback of `Seq.Con.mapJob`'s return value? Why start in parallel and join the result with a promise?